### ISSUE TITLE
Select keyboard navigation should be updated to match dropdown

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#5040]
**Describe the issue. What is the expected and unexpected behavior?**
This is similar to #3955 - we should add the same interaction pattern to select 
When a user clicks on the select toggle with their mouse then presses the down arrow keys, they should be able to navigate through the menu. They should also be able to open the select menu with the down arrow. This also fixes a JAWS issue that was noticed with dropdown that may affect the select component as well.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

Click on the select toggle, then attempt to switch to keyboard navigation

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Bug - JAWS

**What is your product and what release version are you targeting?**
